src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -32,23 +33,20 @@ export class QuerySuggestPreview extends Component implements IComponentBindings
   };
 
   static options: IQuerySuggestPreview = {
+    /**
+     * If you specify no previously registered template through this option, the component uses its default template.
+     */
     resultTemplate: TemplateComponentOptions.buildTemplateOption(),
```

Comment:
This will be combined with the documentation from https://github.com/coveo/search-ui/pull/1272.

Should I add the default template here? @fbeaudoincoveo 
E.g.:

If you specify no previously registered template through this option, the component uses this default template:
```html
<script class="result-template" type="text/html">
  <div class="coveo-result-frame coveo-default-result-preview">
    <div class="CoveoImageFieldValue" data-field="@image"></div>
    <a class="CoveoResultLink"></a>
  </div>
</script>
```

Reply:
I wouldn't include this information in the reference documentation, no. What I'll likely end up doing is include a screen capture or animated GIF in the external documentation to show what the component's default behavior when no custom template is provided.